install/install.sh

```diff
@@ -55,7 +58,7 @@ echo "-------------------------------"
 echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
-read -e -r -p -s "Password of the 'root' user of MySQL: " -i "" db_root_password
+read -e -r -p  "Password of the 'root' user of MySQL: " -i "" db_root_password
```

Comment:
Why remove the -s? It's needed to not echo the input back, which is on purpose in this case.

Reply:
-s is the reason for the "not a valid identifier" 
 read: `Password of the 'root' user of MySQL: ': not a valid identifier
which in turn blocks the input

Reply:
sending a new PR with corrections